### DISCUSSION TITLE
Apparent clock jumps ?

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#979] [Q&A] [answered]
Sometimes the clock seems to be delayed. I first noticed it when studying circuits with D flip flops, but it generally seems to happen when an input makes a transition at the same moment as the clock signal goes up or down. See e.g. the picture below as an illustration in the minimal case where I have just an input and a clock with no circuit attached. Why does it happen ?  What can be done to avoid it ?
<img width="594" alt="Example of clock jumps" src="https://user-images.githubusercontent.com/108168531/178132165-7fd881a0-8b4b-430a-a1eb-5d055df02957.png">

**Comment by hneemann** [comment, marked as answer]
Please see #359

**Reply by jongena** [reply]
Many thanks for the very quick answer, all is understood ! 
With best regards, 

Jef Ongena
